package.json

```
@@ -1,48 +1,48 @@
 {
-	"name": "docusaurus-2-test",
-	"version": "0.0.0",
-	"description": "Documentation for the Telestion project, created using Docusaurus v2",
-	"private": true,
-	"scripts": {
-		"docusaurus": "docusaurus",
-		"start": "docusaurus start",
-		"build": "docusaurus build",
-		"swizzle": "docusaurus swizzle",
-		"deploy": "docusaurus deploy",
-		"clear": "docusaurus clear",
-		"serve": "docusaurus serve",
-		"write-translations": "docusaurus write-translations",
-		"write-heading-ids": "docusaurus write-heading-ids",
-		"prettier:fix": "prettier --write .",
-		"prettier:check": "prettier --check ."
-	},
-	"dependencies": {
-		"@cmfcmf/docusaurus-search-local": "^0.6.5",
-		"@docusaurus/core": "2.0.0-beta.4",
-		"@docusaurus/preset-classic": "2.0.0-beta.4",
-		"@mdx-js/react": "^1.6.21",
-		"@svgr/webpack": "^5.5.0",
-		"clsx": "^1.1.1",
-		"file-loader": "^6.2.0",
-		"mime": "^2.5.2",
-		"prism-react-renderer": "^1.2.1",
-		"react": "^17.0.1",
-		"react-dom": "^17.0.1",
-		"url-loader": "^4.1.1"
-	},
-	"browserslist": {
-		"production": [
-			">0.5%",
-			"not dead",
-			"not op_mini all"
-		],
-		"development": [
-			"last 1 chrome version",
-			"last 1 firefox version",
-			"last 1 safari version"
-		]
-	},
-	"devDependencies": {
-		"prettier": "^2.3.2"
-	}
+  "name": "docusaurus-2-test",
```

What's about a really nice package name :stuck_out_tongue_winking_eye:?

```suggestion
  "name": "telestion-docs",
```